source/stylesheets/base/_feature_sections.scss

```diff
@@ -52,6 +52,40 @@
   }
 }
 
+.drawn-rule {
+  margin: $base-spacing;
+  text-align: center;
+}
+
+.featured-tip {
+  @include clearfix;
+
+  .container {
+    display: flex;
+    justify-content: stretch;
+    padding: $base-spacing;
+
+    .left {
+      @include grid-column(7 of 12);
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-end;
+    }
+
+    .right {
+      @include grid-column(5 of 12);
+      align-items: center;
+      display: flex;
+
+      img {
```

Comment:
Your nesting is getting pretty deep here. As a rule of thumb you don't want to go more the 3 levels deep. It starts to get confusing afterwards. I'd really try and keep it as shallow as possible.